### DISCUSSION TITLE
While Running through CI getting error like  "CypressError: Timed out after waiting `120000ms` for your remote page to load".

### DISCUSSION THREAD

**Discussion by ghost** [cypress-io/cypress#27220] [Questions and Help]
I am using Jenkins to run my cypress scripts and using docker in Jenkins execute shell Comand like I am pulling docker image (docker pull cypress/included:12.16.0) and running docker in execute shell (docker run -v $(pwd):/e2e -w /e2e cypress/included:12.16.0 cypress run --env date="2023-07-06" --spec "cypress/integration/exapmles/BDD/xyz.feature")

After pageloadtimeout it is again throughing error like below:

CypressError: Timed out after waiting `120000ms` for your remote page to load.

Your page did not fire its `load` event within `120000ms`.

You can try increasing the `pageLoadTimeout` value in `cypress.config.js` to wait longer.

Browsers will not fire the `load` event until all stylesheets and scripts are done downloading.

When this `load` event occurs, Cypress will continue running commands.
![Screenshot (160)](https://github.com/cypress-io/cypress/assets/92353261/b2460afc-392a-4749-adc4-869bbc8af7d7)

**Comment by varunraorao** [comment]
i am facing the similar issue

**Comment by Priya143-art** [comment]
Did  you get any solution for this?
